task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -908,10 +922,7 @@ def supervise(
         # If we are told to write logs to a file, redirect the task logger to it.
         from airflow.sdk.log import init_log_file, logging_processors
 
-        try:
-            log_file = init_log_file(log_path)
-        except OSError as e:
-            log.warning("OSError while changing ownership of the log file. ", e)
```

Comment:
This was just redundant as `init_log_file` already capture and warns it:

https://github.com/apache/airflow/blob/6844cce95e1d02b06a2de7159db9eb2ea48ac74b/task_sdk/src/airflow/sdk/log.py#L427-L434

<img width="1061" alt="image" src="https://github.com/user-attachments/assets/22abfe6e-6ef0-4faa-be4c-66399a30be9c" />